src/modelinstance.cpp

```diff
@@ -504,15 +485,10 @@ Status ModelInstance::reloadModel(const ModelConfig& config, const DynamicModelP
 }
 
 Status ModelInstance::recoverFromReloadingError(const Status& status) {
-    if (status == StatusCode::RESHAPE_ERROR) {
-        auto recoveryStatus = this->recoverFromReshapeError();
-        if (!recoveryStatus.ok()) {
-            return recoveryStatus;
-        }
-        return status;
-    }
     SPDLOG_WARN("Failed to reload model: {} version: {} with error: {}. Reloading to previous configuration",
         getName(), getVersion(), status.string());
+    unloadModel();
```

Comment:
All other infer requests will get ret code MODEL_NOT_LOADED_ANYMORE in such case as model will transition through the UNLOADING & END state.

This may be acceptable as a temporary workaround tough.